### ISSUE TITLE
IE11 / win7: inputs do not trigger input and change events

### ISSUE THREAD

**Issue by gr2m** [gr2m/bootstrap-expandable-input#6]
To reproduce

1. open http://gr2m.github.io/bootstrap-expandable-input/
2. open console
3. add event handlers:

```js
$('p[contenteditable]').on('change', function() { console.log ('change!') })
$('p[contenteditable]').on('input', function() { console.log ('input!') })
```

4. type something into the multiline textarea example
5. click somewhere on the page so the input looses focus

### Expected behavior:

You should see `input!` and `change!` in your console

### Actual behavior

No logs in the console at all :crying_cat_face:

**Comment by gr2m** [comment]
It might be worth looking into how https://github.com/upfrontIO/editable.js is doing the event binding here

**Comment by gr2m** [comment]
The input 'event' is not supported in IE on contenteditable. I guess for IE, `keyup` event could be used, see http://stackoverflow.com/questions/1391278/contenteditable-change-events

The `change` event can then be simulated I guess?

**Comment by zoepage** [comment]
this might help:
https://github.com/upfrontIO/editable.js/blob/master/editable.js#L5540

**Comment by gr2m** [comment]
@zoepage not really, that is related to when text changes that is currently selected. What we need is to polyfill `change` and `input` events, I think we could do that by listening to `keyup` and `blur`

**Comment by gr2m** [comment]
I've looked into this today again as another project was affected. We could, in theory, trigger a manual `input` event for IE, but I think I'd rather declare is as known issue, and ask people to use `keyup`, which works just fine here. What do you think @zoepage

**Comment by zoepage** [comment]
Yes, you're right. I think `keyup` is definitely the better option.
